routes/authentication.js

```diff
@@ -0,0 +1,51 @@
+var express = require('express'),
+    bcrypt = require('bcrypt'),
+    db = require('../models'),
+    router = express.Router();
+
+router.get('/register', (req, res) => {
+  if (req.session.user) {
+    res.redirect('/admin/posts');
+  }
+
+  res.render('users/new');
+});
+
+router.post('/users', (req, res) => {
+  db.User.create(req.body).then((user) => {
+    req.session.user = user;
+    res.redirect('/');
+  }).catch(() => {
+    res.redirect('/register');
+  });
+});
+
+router.get('/login', (req, res) => {
+  res.redirect('/admin');
+});
+
+router.post('/login', (req, res) => {
+  db.User.findOne({
+    where: {
+      email: req.body.email
+    }
+  }).then((userInDB) => {
+    bcrypt.compare(req.body.password, userInDB.passwordDigest, (error, result) => {
+      if (result) {
+        req.session.user = userInDB;
+        res.redirect('/');
+      } else {
+        res.redirect('/login');
+      }
+    });
+  }).catch(() => {
```

Comment:
You are right, we could. I didn't pass it on purpose because we are not using that error parameter.